elcord.el

```diff
@@ -43,6 +43,11 @@ See <https://discordapp.com/developers/applications/me>."
                  (function :tag "Call the function with no args to get the ID."))
   :group 'elcord)
 
+(defcustom elcord--refresh-rate 15
```

Comment:
Ditto as in #21 : Can you make this have a single hyphen? Internal functions/variables should be double-hyphened, but 'public' ones should have a single hyphen.
Thanks!

Reply:
Oh yeah sorry, I'll get on it now.